src/components/Search.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { searchPhoto } from "../services/actions";
+
+import "../style/Search.css";
+
+const mapDispatchToProps = { searchPhoto };
+
+const Search = ({ searchPhoto }) => {
+  return (
+    <div className="search-container">
+      <input
+        type="text"
+        placeholder="🔍メンバー名またはテーマ名で検索する"
+        onChange={e => searchPhoto(e.target.value)}
```

Comment:
This is OK for now. But after we move to back-end API or after the data becomes larger, we might need to debounce here.

Reply:
Sure, please elaborate when you have time :)